src/observable/BoundCallbackObservable.ts

```diff
@@ -77,9 +77,9 @@ export class BoundCallbackObservable<T> extends Observable<T> {
     };
   }
 
-  constructor(private callbackFunc: Function,
-              private selector: Function,
-              private args: any[],
+  constructor(public callbackFunc: Function,
```

Comment:
or maybe we can make `dispatch` static method in `boundcallbackObservable`? I would prefer to not expose these values as public.